### DISCUSSION TITLE
How to avoid "Scope already initialized"

### DISCUSSION THREAD

**Discussion by CMathgician** [apideck-libraries/postman-to-k6#50] [Q&A]
Is there anyway/command to import `/libs/shim/core.js`once?
<img width="1363" alt="image" src="https://user-images.githubusercontent.com/6632239/158530699-fb6f59b4-aa82-499e-a9f8-00d2256c144e.png">

**Comment by thim81** [comment]
@CMathgician 

Would you be able to create a copy of the Postman collection but change all sensitive properties?
We are using postman-to-k6 all the time, across multiple teams and we have not run into this issue, so I want to figure what is being generated so that we can fix this for you and all other users.

**Reply by CMathgician** [reply]
Here is [the postman collection](https://drive.google.com/drive/folders/14lOTxkhUSwtpNPefrYecoQsyA9Ijdru5?usp=sharing) that I use.
so I have 2 types of endpoint `internal` dan `public`. Right now it's 2 diff collections.

---
```
import internalScript from "../svc-srs/internal-script.js";
import publicScript from "../svc-sms/public-script.js";
```
Why I'm doing this way. So the idea is I would like to create a smoke test and just call the function/module.

**Comment by thim81** [comment]
Can you perhaps give some context what the 2 files contain?

```
import internalScript from "../svc-srs/internal-script.js";
import publicScript from "../svc-sms/public-script.js";
```
